src/main/scala/com/fulcrumgenomics/sv/tools/SvPileup.scala

```diff
@@ -178,15 +191,11 @@ object SvPileup extends LazyLogging {
 
     if (segments.length == 1) IndexedSeq.empty else {
       segments.iterator.sliding(2).flatMap { case Seq(seg1, seg2) =>
-        var result: Option[PutativeBreakpoint] = determineInterContigBreakpiont(seg1=seg1, seg2=seg2)
-        if (result.isEmpty) {
-          val maxInnerDistance = if (seg1.origin != seg2.origin) maxAlignedSegmentInnerDistance else maxReadPairInnerDistance
-          result = determineIntraContigBreakpiont(seg1=seg1, seg2=seg2, maxInnerDistance=maxInnerDistance)
-        }
-        if (result.isEmpty) result = determineOddPairOrientation(seg1=seg1, seg2=seg2)
+        var result                 = determineInterContigBreakpoint(seg1, seg2)
+        if (result.isEmpty) result = determineIntraContigBreakpoint(seg1, seg2, maxAlignedSegmentInnerDistance , maxReadPairInnerDistance)
+        if (result.isEmpty) result = determineOddPairOrientation(seg1, seg2)
```

Comment:
Shall we collect all evidence for this template, or just the first one?  I have it just the first one to make the summary statistics for each breakpoint have `sum(# of templates) == sum(# of evidences)`.  For annotating reads, I think I want to know all types of evidence that template implies though.

Reply:
I think given our discussion, we're in a good place.  We'll reduce evidence types down to `SplitRead` and `ReadPair` and that'll be it.